File: problem2A1.py
import numpy as np
	
l=int(input("Enter length of array "))
arr=[]
for i in range(0,l):
	a=float(input('Enter element '))
	arr.append(a)
n=int(input('Enter n(for top n)'))
for t in range(0,n):
	g=arr[t]
	j=t
	for i in range(t,l):
		if g<arr[i]:
			g=arr[i]
			j=i
	s=arr[j]
	arr[j]=arr[t]
	arr[t]=s
for t in range(0,n):
	print(arr[t])

File: problem2A1.cc
#include<iostream>
using namespace std;
int main()
{	
	int l,j,n;
	cout<<"Enter length of array";
	cin>>l;
	cout<<"Enter n(for top n values): ";
	cin>>n;
	float arr[l],b,s;
	for(int i=0;i<l;i++)
	{
		cout<<"Enter element no "<<i+1<<" of array"<<endl;
		cin>>arr[i];
		
	}
	for(int t=0;t<n;t++)
	{
		b=arr[t];
		j=t;
		for(int i=t;i<l;i++)
		{
			if(b<arr[i])
			{	
				b=arr[i];
				j=i;
			}
		}
		s=arr[j];
		arr[j]=arr[t];
		arr[t]=s;
	}
	for(int i=0;i<n;i++)
	{
		cout<<arr[i]<<" ";
	}
}
